pkg/api/controller_test.go

```diff
@@ -234,8 +236,51 @@ func TestCreateRepoDBDriver(t *testing.T) {
 			"versiontablename":      1,
 		}
 
-		testFunc = func() { _, _ = api.CreateRepoDBDriver(conf.Storage.StorageConfig, log) }
+		testFunc = func() { _, _ = repodbfactory.New(conf.Storage.StorageConfig, log) }
 		So(testFunc, ShouldPanic)
+
```

Comment:
For the record these tests will be moved under pkg/meta in the next iteration/pr.